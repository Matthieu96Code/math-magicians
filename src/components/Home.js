import styles from '../styles/Body.module.css';

const Home = () => (
  <div className={styles.homePage}>
    <h2>Welcome to our page!</h2>

    <p>
      Here, we aim to make math a fun and enjoyable experience for everyone.
      Whether you&apos;re a student struggling with algebra or a math enthusiast looking for a
      challenge, our website has something for you.
    </p>

    <p>
      One of the main features of our website is our calculator.
      Our calculator is designed to handle a wide range of mathematical functions,
      from basic arithmetic to complex calculus equations.
      It&apos;s user-friendly and can help you solve problems quickly and accurately.
    </p>

    <p>
      In addiion to our calculator, we also have a section on our website
      dedicated to math quotes. We believe that math can be inspirational
      and thought-provoking, and we&apos;ve gathered some of the best quotes from
      mathematicians throughout history to share with our visitors. These quotes
      are perfect for anyone looking for a little extra motivation or
      inspiration to tackle a difficult math problem.
    </p>

    <p>
      So whether you&apos;re looking to solve a tricky math equation or just need
      a little inspiration, the Math Magician website is here to help. Thank
      you for visiting, and we hope you enjoy your time here!
    </p>
  </div>
);
export default Home;
