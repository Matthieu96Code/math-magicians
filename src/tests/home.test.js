import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders correctly', () => {
  const component = renderer.create(
    <Home />,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
