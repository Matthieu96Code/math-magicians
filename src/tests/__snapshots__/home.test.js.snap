// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="homePage"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Here, we aim to make math a fun and enjoyable experience for everyone. Whether you're a student struggling with algebra or a math enthusiast looking for a challenge, our website has something for you.
  </p>
  <p>
    One of the main features of our website is our calculator. Our calculator is designed to handle basic arithmetic functions. It's user-friendly and can help you solve problems quickly and accurately.
  </p>
  <p>
    In addiion to our calculator, we also have a section on our website dedicated to happiness quotes. We believe that math can be inspirational and thought-provoking, and we've gathered some of the best quotes to share with our visitors. These quotes are perfect for anyone looking for a little extra motivation or inspiration to tackle a difficult math problem.
  </p>
  <p>
    So whether you're looking to solve a tricky math equation or just need a little inspiration, the Math Magician website is here to help. Thank you for visiting, and we hope you enjoy your time here!
  </p>
</div>
`;
